Makefile

```diff
@@ -44,12 +44,18 @@ cluster/prepare:
 	-kubectl create -n $(NAMESPACE) -f deploy/service_account.yaml
 	-kubectl create -n $(NAMESPACE) -f deploy/role.yaml
 	-kubectl create -n $(NAMESPACE) -f deploy/role_binding.yaml
-	-kubectl apply -f deploy/crds/push_v1alpha1_unifiedpushserver_crd.yaml
+	-kubectl apply -n $(NAMESPACE) -f deploy/crds/push_v1alpha1_unifiedpushserver_crd.yaml
+	-kubectl apply -n $(NAMESPACE) -f deploy/crds/push_v1alpha1_pushapplication_crd.yaml
+	-kubectl apply -n $(NAMESPACE) -f deploy/crds/push_v1alpha1_androidvariant_crd.yaml
+	-kubectl apply -n $(NAMESPACE) -f deploy/crds/push_v1alpha1_iosvariant_crd.yaml
 
 .PHONY: cluster/clean
 cluster/clean:
 	-kubectl delete -f deploy/role.yaml
```

Comment:
Because there are now finalizers on the PushApplication, AndroidVariant, and IOSVariant CRs, those will have to be cleaned up here first before anything else (otherwise manual intervention will be needed to delete them to allow the namespace to be deleted).

Thanks to @psturc for highlighting that. I'll add it now.

Reply:
Done in latest push.

Reply:
👍 works perfectly